### PR TITLE
fixes #18250 - show compute resource xhr errors

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

**Comment by ohadlevy** [comment]
@timogoebel do you mind adding a couple of screenshots before / after this change? thanks!

**Comment by timogoebel** [comment]
@ohadlevy: Sure.

Before:
image

After:
image

File: app/assets/javascripts/compute_resource.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ $(function() {
     var url = $(this).attr('data-url');
     $(this).load(url + ' table', function(response, status, xhr) {
       if (status == "error") {
-        $(this).closest(".tab-content").find("#spinner").html(Jed.sprintf(__("There was an error listing VMs: %(status)s %(statusText)s"), {status: xhr.status, statusText: xhr.statusText}));
+        $(this).html(Jed.sprintf(__("There was an error listing VMs: %(status)s %(statusText)s"), {status: xhr.status, statusText: xhr.statusText}));
```
Comment (ohadlevy) [inline app/assets/javascripts/compute_resource.js:7]: @timogoebel thanks, the image looks good, why did you remove the finder? (wasnt required?)

Comment (timogoebel) [reply app/assets/javascripts/compute_resource.js:7]: `closest(".tab-content")` does actually exist, but is "further up" the DOM tree than `$(this)`. `$(this)` actually refers to the tab-pane. `#spinner` does not exist at all (and `id` should be unique on the page btw).

Comment (timogoebel) [reply app/assets/javascripts/compute_resource.js:7]: To further illustrate:
image

**Comment by ohadlevy** [comment]
thanks @timogoebel
